sandesh/library/c/test/SConscript

```diff
@@ -11,7 +11,7 @@ env = SandeshEnv.Clone();
 
 env.Append(CPPPATH = [
         Dir('#build/include').abspath,
-        Dir('#controller/src').abspath,
+#        Dir('#controller/src').abspath,
```

Comment:
Why?

First, why is this entry not needed any more?

And if it's not needed, let's delete it properly instead of leaving commented-out code that causes head-scratches later.